remotespark/livyclientlib/pandaslivyclientbase.py

```diff
@@ -28,10 +30,11 @@ def _execute_dataframe_helper(self, context_name, command):
                 return self.get_data_dataframe(records_text)
         except (ValueError, SyntaxError):
             self.logger.error("Could not convert sql results to pandas DF.")
-            return records_text
+            raise DataFrameParseException(records_text)
 
     def get_columns(self, context_name, command):
-        return self.execute(self.make_context_columns(context_name, command))
+        return self.execute(self.make_context_columns(context_name,
```

Comment:
Will the success/failure here cause a ValueError or SyntaxError (except statement in line 31 above)?